.storybook/preview.js

```diff
@@ -0,0 +1,13 @@
+import {StyledEngineProvider} from '@mui/material/styles';
```

Comment:
can we setup eslint for storybook?

Reply:
I think yes. But I would do it in the next prs